docs/reference/compatibility/edot-vs-upstream.md

```diff
@@ -0,0 +1,39 @@
+---
+navigation_title: EDOT compared to upstream OTel
+description: Differences between Elastic Distributions of OpenTelemetry (EDOT) and upstream OpenTelemetry.
+applies_to:
+  stack:
+  serverless:
+    observability:
+products:
+  - id: cloud-serverless
+  - id: observability
+---
+
+# EDOT compared to upstream OpenTelemetry
+
+The [Elastic Distributions of OpenTelemetry (EDOT)](../index.md) are based on the upstream OpenTelemetry project but include additional features and configurations that are specific to the Elastic ecosystem. Each EDOT component is carefully selected and tested to ensure it works seamlessly with Elastic Stack components.
+
+Here are some key differences and considerations when using EDOT compared to upstream OpenTelemetry:
+
+| Feature | EDOT | Upstream OpenTelemetry |
+|---------|------|------------------------|
+| Configuration | Configured for Elastic Observability. | Requires manual configuration.|
+| Support | Official Elastic support with SLAs. | Community support only. |
+| Integration | Seamless integration with Elastic Stack. | Requires additional configuration for Elastic. |
+| Components | Curated list of components for Elastic Observability. | Generic components that may not support all Elastic features. |
+| Deployment | Easier to deploy with Elastic Stack. | Requires manual setup and configuration. |
+| Compatibility | Fully compatible with Elastic Stack components. | Compatible but may require additional configuration. |
+| Self-managed/ECH | Required for full functionality | Compatible but without guaranteed support |
+| Updates | Future updates aligned with Elastic Stack releases. | Updates depend on upstream OpenTelemetry release cycle. |
+| Custom components | Includes custom components optimized for Elastic. | Uses standard OpenTelemetry components. |
+
+EDOT offers a streamlined experience with less configuration burden compared to upstream OpenTelemetry. While you can use upstream components with Elastic, these components aren't covered under official Elastic support SLAs.
```

Comment:
Would it be helpful to link to https://www.elastic.co/support/welcome in this paragraph?

Reply:
Good point! I wonder if support boilerplate could be included by default in all docs' footers...